nixos/modules/services/misc/paperless.nix

```diff
@@ -378,5 +418,33 @@ in
         gid = config.ids.gids.paperless;
       };
     };
-  };
+  }) (mkIf (cfg.enable && cfg.backup.enable) {
+
+    systemd.tmpfiles.rules = [
+      "d '${cfg.backup.directory}' - ${cfg.user} ${config.users.users.${cfg.user}.group} - -"
+    ];
+
+    systemd.timers.paperless-backup = {
+      description = "paperless backup timer";
+      wantedBy = [ "timers.target" ];
+      timerConfig = {
+        OnCalendar = cfg.backup.calendar;
+        AccuracySec = "5m";
```

Comment:
What is the reason so set this?

Reply:
"inspiratin" from the mysql backup timer. Happy to remove that if you want.